Purchases/Purchasing/StoreKitAbstractions/StoreProductDiscount.swift

```diff
@@ -70,7 +70,9 @@ public final class StoreProductDiscount: NSObject, StoreProductDiscountType {
     @objc public var offerIdentifier: String? { self.discount.offerIdentifier }
     @objc public var currencyCode: String? { self.discount.currencyCode }
     @objc public var price: Decimal { self.discount.price }
+    @objc public var localizedPriceString: String { self.discount.localizedPriceString}
```

Comment:
This needs to go to the APITesters too I think?

Reply:
Maybe I should write code in API testers first and _then_ write the actual code so I don't forget. API Tester driven development?